corehq/apps/reports/standard/cases/basic.py

```diff
@@ -277,7 +277,7 @@ def _case_name(request=None, **context):
         from corehq.apps.reports.views import CaseDataView
         return [
             {
-                'title': _case_name,
+                'title': _get_case_name,
```

Comment:
huh, I didn't realize how ambiguous my previous comment was. I was talking about `'title'` -> `'get_title'` (the key in the dict).

The name of the local function hadn't even entered my mind because the fact that it references a function is clear in the local code context.

However, the returned list of dicts is used in other contexts were it's less clear
```py
for page in CaseListReport.get_subpages():
    page['title']  # <--- this is a function, not a string
```
Maybe the name of that key predates your previous PR?

Reply:
Oh, I see what you meant - the key name didn't cross my mind `title` can be [either a string or a function](https://github.com/dimagi/commcare-hq/blob/master/corehq/tabs/templatetags/menu_tags.py#L126). I don't love values that can be multiple types, but I'm not keen on reworking that logic right now, or on renaming it something verbose like `title_string_or_getter`.

I agree this PR isn't exceptionally useful, but I do like `_get_foo` a little better than `foo`, going to merge it.